src/sknnr/_msn.py

```diff
@@ -24,46 +24,50 @@ class MSNRegressor(YFitMixin, TransformedKNeighborsRegressor):
     ----------
     n_neighbors : int, default=5
         Number of neighbors to use by default for `kneighbors` queries.
-    n_components : int, default=None
-        Number of components to keep during CCorA transformation. If None, all
-        components are kept. If n_components is less than the number of available
+    n_components : int, optional
+        Number of components to keep during CCorA transformation. If `None`, all
+        components are kept. If `n_components` is greater than the number of available
         components, an error will be raised.
     weights : {'uniform', 'distance'} or callable, default='uniform'
         Weight function used in prediction.
     algorithm : {'auto', 'ball_tree', 'kd_tree', 'brute'}, default='auto'
         Algorithm used to compute the nearest neighbors.
     leaf_size : int, default=30
-        Leaf size passed to BallTree or KDTree.
+        Leaf size passed to `BallTree` or `KDTree`.
     p : int, default=2
         Power parameter for the Minkowski metric.
     metric : str or callable, default='minkowski'
-        The distance metric to use for the tree, calculated in CCorA space.
+        The distance metric to use for the tree, calculated in CCA space.
```

Comment:
Ooh, just caught that this should be `CCorA` space.  It looks like you changed it in this commit.  Was there a reason for that?

Reply:
Over-eager copy pasting, thanks for catching it!